### PR TITLE
Add create a tenant doc

### PR DESCRIPTION
**Pull request by TylerMSFT**
Adding a new doc for creating a demo tenant and app object so that you can test out OSA.

### CODE REVIEW

File: docs/howto-create-tenant.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+For now, you will get message that the site is not secure. Click **Details** > **Go on to the webpage**
+
+Log in with the user and password that you created in [Create new user in Active Directory](#create-new-user-in-active-directory)
+When the **Permissions requested** dialog appears, select **Consent on behalf of your organization** and then **Accept**.
+
+You should then see the Openshift container platform catalog in your browser. You now have an Azure Red Hat OpenShift cluster in your tenant.
+
+## Resources
+
```
Comment (sozercan) [inline docs/howto-create-tenant.md]: looks like these are not in new lines

**Comment by TylerMSFT** [comment]
Not sure ‘not in new lines’ means.

**Comment by sozercan** [comment]
All the links are in a single line.

Looks like this:

Screen Shot 2019-03-22 at 12 42 31

File: docs/howto-create-tenant.md (hunk 1 above)
Comment (TylerMSFT) [reply docs/howto-create-tenant.md]: Fixed.  (An annoying markdown thing. Requires two spaces after those lines to generate a carriage return between each)

**Pull request merged**